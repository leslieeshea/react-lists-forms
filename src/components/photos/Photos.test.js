import React from 'react';
import { shallow } from 'enzyme';
import Photos from './Photos';

describe('Photos component', () => {
  it('renders a list of Photos', () => {
    const photos = [
      '/photo1',
      '/photo2',
      '/photo3'
    ];

    const wrapper = shallow(<Photos photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
