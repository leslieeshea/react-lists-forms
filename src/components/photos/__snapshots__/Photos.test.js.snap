// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photos component renders a list of Photos 1`] = `
<ul>
  <li
    key="/photo1"
  >
    <Photo
      photo="/photo1"
    />
  </li>
  <li
    key="/photo2"
  >
    <Photo
      photo="/photo2"
    />
  </li>
  <li
    key="/photo3"
  >
    <Photo
      photo="/photo3"
    />
  </li>
</ul>
`;
